package-lock.json

```diff
@@ -1930,6 +1930,7 @@
 				"jest-resolve": "^26.6.2",
 				"jest-util": "^26.6.2",
 				"jest-worker": "^26.6.2",
+				"node-notifier": "^8.0.0",
```

Comment:
Lgtm 👍, except the package-lock changes here.

Does removing all `node_modules` and rerunning `npm run setup` cut it?

Reply:
Is there any way for me to uncommit the package-lock changes? Removing the node modules and rerunning npm run setup does not fix this.

Reply:
> Removing the node modules and running npm run setup does not fix this.

Sorry, this should follow the `git revert` above. Post back with your environment (npm version, OS, ...) if you're still getting these strange changes after reverting 😬, might be an issue with the dev setup.

Reply:
Can I ask if changing the package-lock file here even matter? Since everyone should be running `npm run setup` on the first pull, Im not sure if this makes a difference. 

But I think that the package-lock file is updated cause I pulled it from the newest merge that someone else did.

Reply:
@ong6 you are mostly right that it dosen't matter from a environment setup pov.
However, It's an unrelated change to this PR there are no dependency edits here.
If it occured for a reason (and it should be found), we should be fixing it in a separate, focused commit and pr for clarity (where do we more specific testing as well -- e.g. on different OS'es)